src/control/server/init/setup_spdk.sh

```diff
@@ -97,11 +97,9 @@ else
 		'/sys/class/uio/uio*/device/config'	\
 		'/sys/class/uio/uio*/device/resource*'; do
 
-		# shellcheck disable=SC2086
-		if list=$(ls -d $glob); then
-			echo -n "RUN: ls -d $glob | xargs -r chown -R "
-			echo "$_TARGET_USER"
-			echo "$list" | xargs -r chown -R "$_TARGET_USER"
+		if compgen -G "${glob}" > /dev/null; then
+			echo "RUN: chown -R ${_TARGET_USER} ${glob}"
+			chown -R "${_TARGET_USER}" "${glob}"
```

Comment:
It should not work: the completion will not be done with "..."
```suggestion
			chown -R "${_TARGET_USER}" "$(compgen -G "{glob}")"
```

Reply:
why should this not be quoted? it will always be a single string

Reply:
Quote will prevent globbing. If `glob` variable is equal to `foo*` then `foo_bar` will not be matched.
However, if you want that `glob` to be strictly equal to the file path, then you are right.
I was understanding that you want globbing to take place ?

Reply:
that's correct , thank you for spotting this, I've just verified on wolf

Reply:
I will also push the fix on master